tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh

```diff
@@ -111,6 +129,68 @@ do
   delim=", "
 done
 
+models_io_json='{ "inputs" : [], "outputs" : [] }'
+for models_conn_path in ${models_conn_src[@]}
+do
+  models_conn_file=$(basename "$models_conn_path")
+
+  if [ ! -e $models_conn_file ]; then
+    >&2 echo "error: "$models_conn_file" does not exist."
+    exit 1
+  fi
+
+  inputs=$(jq -r '.inputs' $models_conn_path)
+  models_io_json=$(echo $models_io_json | jq --argjson ins "$inputs" '.inputs += [$ins]')
+
+  outputs=$(jq -r '.outputs' $models_conn_path)
+  models_io_json=$(echo $models_io_json | jq --argjson outs "$outputs" '.outputs += [$outs]')
+done
+
+declare -A connect_map
+in_model=0
+for model_inputs in $(echo "${models_io_json}" | jq -c '.inputs[]'); do
+  input_pos=0
+
+  for org_input in $(echo "${model_inputs}" | jq '.org[]'); do
+    out_model=0
+
+    for model_outputs in $(echo "${models_io_json}" | jq -c '.outputs[]'); do
+      if [[ $in_model == $out_model ]]; then
+        continue
+      fi
+
+      output_pos=0
+
+      for org_output in $(echo "${model_outputs}" | jq '.org[]'); do
+        if [[ $org_input == $org_output ]]; then
+          to=$(echo "$in_model:0:$input_pos" | jq -R)
+          from=$(echo "$out_model:0:$output_pos" | jq -R)
+          pkg_inputs_str+="$from,"
```

Comment:
`from` is an output of a separate model, is it correct to add it to `pkg-inputs`? I'm a little confused.

Reply:
No. `pkg-inputs`is the user input. The internal connection between models should not be in `pkg-inputs`.

Reply:
Oh, I was misunderstanding. I will fix it.